src/main/java/io/microprofile/config/spi/Converter.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright 2016 Microprofile.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ *******************************************************************************/
+
+package io.microprofile.config.spi;
+/**
+ * <p>A very simple interface for conversion of configuration values from String to any Java type.</p>
+ *
+ * <p>A Converter can specify a {@link javax.annotation.Priority}.
+ * If no priority is explicitly assigned, the value of 100 is assumed.</p>
+ *
+ * <p>If multiple Converter get found the one with the highest priority will be used.</p>
```

Comment:
+1
I suggest the following: "If there are multiple Converters with the same highest priority a ConfigurationException will be thrown"